Objects/call.c

```diff
@@ -1051,6 +1051,27 @@ PyObject_CallMethod(PyObject *obj, const char *name, const char *format, ...)
 }
 
 
+PyAPI_FUNC(PyObject *) PyObject_CallMethodArgs(PyObject *obj,
+                                               const char *name,
+                                               PyObject *args,
+                                               PyObject *kwargs)
+{
+    PyObject *callable, *retval;
+
+    if (obj ==  NULL)
```

Comment:
Please, use braces as indicated in pep 7: https://www.python.org/dev/peps/pep-0007/